cruise-control/src/integrationTest/java/com/linkedin/kafka/cruisecontrol/BrokerFailureIntegrationTest.java

```diff
@@ -47,14 +48,21 @@ public class BrokerFailureIntegrationTest extends CruiseControlIntegrationTestHa
   private static final int BROKER_ID_TO_REMOVE = 1;
 
   private final Boolean _kafkaBrokerFailureDetectorEnable;
+  private final Boolean _vertxEnabled;
 
-  public BrokerFailureIntegrationTest(Boolean kafkaBrokerFailureDetectorEnable) {
+  public BrokerFailureIntegrationTest(Boolean kafkaBrokerFailureDetectorEnable, Boolean vertxEnabled) {
```

Comment:
It looks like we just passed this parameter to the test, but we didn't actually integration-test vertx requests. Otherwuse, could you point me to where do we actually test the REST requests?

Reply:
Well, in VertxAvailabilityIntegrationTest we test if the endpoints are responding or not. In the modified integration test if we turn on `vertx.enabled`, then the app runs with the verx implementations of the interfaces and uses them to complete the request. So we tested if we can access the endpoints and if the logic behind the requests works fine with vertx.
But if I miss something or you would prefer some other way to test it, let me know.